doc/source/releases/20.09_announce_user.rst

```diff
@@ -0,0 +1,184 @@
+
+An example of a Dropbox account:
+
+.. figure:: images/20.09-upload-dropbox.png
+   :alt: Screenshot of Dropbox with a "bams" folder and several test files
+
+There will be a new button for "Choose Remote files", when sources are configured, and then you will be able to see and search across all of the remote data locations to which you might have access.
+
+.. figure:: images/20.09-upload-search.png
+   :alt: Search across sources. A modal dialog showing a new "Dropbox" folder amongst "FTP Directory" and library location.
+
+And then you will be able to see your load your files easily in Galaxy!
+
+.. figure:: images/20.09-upload-dropbox-gx.png
+   :alt: Screenshot of the same files from the Dropbox screenshot, accessible in Galaxy.
+
+The admin will need to configure this feature, and then you will need to provide your credentials for these services in Galaxy's user preferences, if you want to use them. Ask you admin!
+
+
+GTN In Galaxy
+-------------
+
+During the BCC2020 CoFest, `@hexylena <https://github.com/hexylena>`__ and `@shiltemann <https://github.com/shiltemann>`__ worked together to add support for accessing `Galaxy Training Network <https://training.galaxyproject.org>`__ tutorials from within Galaxy. The graduation cap icon will activate the interface.
+
+For updated tutorials, tools will be highlighted as blue buttons. When clicked, these buttons will hide the GTN and take you directly to the correct version of the correct tool within the Galaxy interface. No more time spent figuring out which "cut" tool the tutorial author meant! And when you're ready, you can easily jump back to the training materials.
+
+.. raw:: html
+
+   <video controls autoplay loop muted width="100%">
+       <source src="../_static/20.09-gtn-in-galaxy.mp4" type="video/mp4">
+   </video>
+
+Thanks `@pvanheus <https://github.com/pvanheus>`__ for the video of it in action!
+
+Upload Directly from the Tool From
+----------------------------------
+
+Did you ever start configuring a tool but forgot to upload your datasets first? And had to start over? Now you can upload datasets directly in the tool form thanks to `@jmchilton <https://github.com/jmchilton>`__. `Pull Request 9809`_
+
+.. raw:: html
+
+   <video controls autoplay loop muted width="100%">
+       <source src="../_static/20.09-upload-form.mp4" type="video/mp4">
+   </video>
+
+
+Improved Workflow Layout Algorithm
+----------------------------------
+
+The old automatic workflow layout algorithm liked to hide the flow of data within your workflow
+
+.. figure:: images/20.09-wf-before.png
+   :alt: Screenshot of a large workflow in Galaxy. Many connections go behind tools and are hard to follow.
+
+
+So we have replaced the layout algorithm with a newer one which should make the flow of data within your workflows easier to understand. You will still probably want to make manualy adjustments after the automatic layout, but hopefully this will save you some time.
```

Comment:
```suggestion
So we have replaced the layout algorithm with a newer one which should make the flow of data within your workflows easier to understand. You will still probably want to make manual adjustments after the automatic layout, but hopefully this will save you some time.
```